articles/private-5g-core/azure-private-5g-core-release-notes-2303.md

```diff
@@ -48,6 +48,7 @@ The following table provides a summary of known issues carried over from the pre
   | 3 | Install/upgrade | Changing the technology type of a deployment from 4G (EPC) to 5G using upgrade or site delete and add is not supported. | Please contact support for the required steps to change the technology type. |
   | 4 | Packet forwarding | In scenarios of sustained high load (for example, continuous setup of 100s of TCP flows per second) combined with NAT pinhole exhaustion, AP5GC can encounter a memory leak, leading to a short period of service disruption resulting in some call failures. | In most cases, the system will recover automatically and be ready to handle new requests after a few seconds' disruption. UEs will need to re-establish any dropped connections. |
   | 5 | Install/Upgrade | In some cases, the packet core reports successful installation even when the underlying platform or networking is misconfigured. | Not applicable. |
+  | 6 | AAD and Web Proxy | Where Azure Active Directory is used to authenticate access to Grafana/SAS, this traffic does not transmit via the Web Proxy when enabled on the ASE (Azure Stack Edge) appliance that the packet core is running on. |
 
```

Comment:
Should include in here that install will fail if the ASE is behind a firewall which blocks traffic that does not go via the proxy.

Also include mitigation steps (disable AAD)